papermill/tests/test_execute.py

```diff
@@ -111,16 +111,17 @@ def test_double_backslash_quote_params(self):
         self.assertEqual(test_nb.parameters, {'foo': r'\\"bar\\"'})
 
     def test_prepare_only(self):
-        path = get_notebook_path('broken1.ipynb')
-        result_path = os.path.join(self.test_dir, 'broken1.ipynb')
-        # Should not raise as we don't execute the notebook at all
-        execute_notebook(path, result_path, {'foo': r'do\ not\ crash'}, prepare_only=True)
-        nb = read_notebook(result_path)
-        self.assertEqual(nb.node.cells[0].cell_type, "code")
-        self.assertEqual(
-            nb.node.cells[0].get('source').split('\n'),
-            ['# Parameters', r'foo = "do\\ not\\ crash"', ''],
-        )
+        for example in ['broken1.ipynb', 'keyboard_interrupt.ipynb']:
```

Comment:
If we were using free standing test functions I would suggest we use https://docs.pytest.org/en/latest/parametrize.html to move the iteration to the outside. That way we'd get two tests (one for each notebook) and see which one it is that failed. Does anyone know how to do that with this style of test setup?